Task91/Task91/Program.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task91
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите целое число, не равное 0");
+            int b = int.Parse(Console.ReadLine());
+            var check = true;
+            while (check)
+            {
+                check = b == 0;
+                if (!check)
+                    break;
+                Console.WriteLine("Введите верное значение");
+                b = int.Parse(Console.ReadLine());
+            }
+            int summ = 0;
+            for (int i = -b; i <= b * b; i++)
+            {
+                summ += i * i;
+            }
+            Console.WriteLine($"Сумма квадратов всех целых чисел от -{b} до ({b} в квадрате) = {summ}");
```

Comment:
1. Здесь больше подходит цикл `do-while`, т. к. необходимо, чтобы тело цикла было выполнено хотя бы один раз. Тогда можно было обойтись без флага `check`.
2. Отделяйте смысловые участки кода, например операторы условия или циклы пустыми строками. Так код читается легче.
3. Если тело цикла состоит из одной строки, то `{ }` можно не использовать.
4. В сообщении печати верхнюю границу изменения `b` лучше печатать числом, а не словами.
```suggestion         
            int b;
            
            do
            {
                Console.WriteLine("Введите целое число, не равное 0");
                b = int.Parse(Console.ReadLine());                
            } while (b == 0);
            
            int summ = 0;
            
            for (int i = -b; i <= b * b; i++) 
                summ += i * i;
            
            Console.WriteLine($"Сумма квадратов всех целых чисел от -{b} до {b * b} = {summ}");
```